src/Components/FlashBanner.tsx

```diff
@@ -0,0 +1,44 @@
+import React from "react"
+import { Banner, Sans } from "@artsy/palette"
+
+interface Props {
+  messageCode: FlashMessageKey
+}
+
+export type FlashMessageKey =
+  /* email confirmed */
+  | "confirmed"
+  /* email already confirmed */
+  | "already_confirmed"
+  /* email confirmation token was invalid */
+  | "invalid_token"
+  /* email confirmation token was empty */
+  | "blank_token"
+  /* email confirmation token was expired */
+  | "expired_token"
+
+type BannerMessage = string | React.FC
+
+const messages: Record<FlashMessageKey, BannerMessage> = {
+  confirmed: "Your email has been confirmed.",
+  already_confirmed: "You have already confirmed your email.",
+  invalid_token: "Your token is invalid. Please try again.",
+  blank_token: "No token found. Please try again.",
+  expired_token: "An error has occurred. Please try again.",
```

Comment:
Shouldn't this be something about how their confirmation token expired, not the general message? Or are we planning to come back to this once we allow people to request the email?

Reply:
I copied these from the [eigen pr](https://github.com/artsy/eigen/pull/3325/files#diff-2996077a298cdc9e41e9bf32fc2431d1R66)- happy to change but also wonder if these are meant to be final - in most cases for example 'try again' is not going to be actionable unless there is a 'resend email' button.

Reply:
Agreed – would much rather tell people to "Contact support@artsymail.com"